types/vote_test.go

```diff
@@ -47,9 +48,11 @@ func exampleVote(t byte) *Vote {
 
 func TestVoteSignable(t *testing.T) {
```

Comment:
Ah nice, the test-vectors are still here 👍 
IMO anything consensus critical should have test-vectors and these test-vectors should live in the spec (or at least should be referenced in the spec).

Reply:
I will add specific ones for canonical. I can also add them for msgs in most reactors. Best to be safe